.gitignore

```diff
@@ -1 +1,6 @@
 /build
+/.vs
```

Comment:
can we please remove these IDE-specific entries? I think they rather belong in a users `core.excludesfile`.

Reply:
Sure! It just made my life easier. I don't know what `core.excludesfile` is but I will have to google it.

Reply:
Done!